### ISSUE TITLE
Alt Launcher fixes and suggestions

### ISSUE THREAD

**Issue by Python-Exoproject** [exoscoriae/eXoDOS#4874]
I discovered that when you install a game using the altlauncher instead of the main launch option, when the game has finished installing and it runs the first time it isnt using the selected altlauncher shader. On a second launch it works as intended.

I investigated and traced it to the last minute change made that meant instead of setting the glshader back to empty using ssr we changed to having a backup of the file in its starting state thats copied in at the beginning of the altlauncher bat.

The process implemented works, but the issue is that the check to see if the game is installed happens first and the install loop returns to a point after. So any time the alt launcher is used to install a game the part that sets the conf back to starting state is skipped.

Fix is easy:

**:start**
IF NOT EXIST ".\eXoDOS\%GameDir%\" goto none

at the top and

:yes
cd %VAR%
call install.bat
cd ..
.\util\setconsole.exe /minimize
goto **start**
:no
goto end

but this will need to be rolled out to all alt launchers in the pack which means its probably not something we want to roll out in an update as the bugs impact on the user is pretty minor (just run the alt launcher a second time after install or install with main launcher and its fine) compared to the effect on using the updater to replace every single alt launcher.

See the third comment for a copy of alt launcher with this change and another that you may want to include

**Comment by Python-Exoproject** [comment]
In a related note the conf we add to the stack for Pixel Perfect mode when the user makes that choice needs the path to the default shader updated. Since staging team now default Staging to the auto crt shader they moved the old default into the subfolder 'interpolation'. Fixed conf here: [Staging_PP.zip](https://github.com/exoscoriae/eXoDOS/files/13627521/Staging_PP.zip)

**Comment by Python-Exoproject** [comment]
Another thought while we are looking at the alt launcher again. Should we consider a choice to just run the game in the alt launcher without the shader\pixel perfect choices on? 

A good number of people are using the alt launcher as a pathway to play the games using Staging even when we dont have them setup to do so from the main launch which is one of the reasons the alt launcher exists (with the proviso that it may not work) but we force them to do something fancy like use pixel perfect or the shader. 

To assist those that want to do nothing fancy but want to use Staging due to its superior sound emulation for example and we havent set it up via Staging I think this option would be appreciated.

**Comment by Python-Exoproject** [comment]
In a recent discussion the idea was raised to make the alternate launcher work like the installer with a stub file with the game files pointing to a single file that has the actual script

**Comment by exoscoriae** [comment]
the first change has been made.

I also implemted the conf file changes from the second comment. that is good info.

I am reluctant to take action on the third comment. I'd rather just start converting games to staging, and ditch vanilla 0.74.

As we plan on doing this anyways, it would make the alt launcher choice redundant.

Which then means going through and changing 8,000 files again to remove that option.

edit: ok, I saw the stub suggestion and implemented it

So, this is nice, as it means no longer making 8,000 changes anymore when we tweak stuff here. hurrah!

I suppose if we really want to make it so that staging can be run without pixel perfect, we could.... but why? Pixel perfect looks so nice! lol
